[Improvement]: pimcore:bundle:install should no-op if a bundle is already installed

### Improvement description

Currently, when running `bin/console pimcore:bundle:install` with a bundle that is already installed, an error is generated: `[ERROR] Bundle BundleName can not be installed.` This is a problem in Docker-based and continuous integration/deployment scenarios, as we regularly run (and re-run) such commands as part of the init operations of our applications. In order to workaround this issue, we have to use the `--fail-without-error` flag (which may ignore legitimate problems with the bundle installation), or choose to ignore errors altogether in the init script (e.g. by _not_ setting `set -e` in the entrypoint of a container, which is not ideal as this will cause the script to skip other legitimate errors)

In my view, trying to install a bundle that is already installed is not an error situation, and the default behaviour of this command should be to no-op and continue without error. **Alternatively**, if the behaviour of the current command is desirable, I suggest adding an optional flag to the command to ignore bundles that are already installed.

--- a/bundles/CoreBundle/src/Command/Bundle/InstallCommand.php
+++ b/bundles/CoreBundle/src/Command/Bundle/InstallCommand.php
@@ -49,7 +49,7 @@ class InstallCommand extends AbstractBundleCommand
     {
         $bundle = $this->getBundle();
 
-        if (!$this->bundleManager->canBeInstalled($bundle) || $this->bundleManager->isInstalled($bundle)) {
+        if ($this->bundleManager->isInstalled($bundle)) {
             $this->io->success(sprintf('Bundle "%s" is already installed', $bundle->getName()));
             return 0;
         }

--- a/bundles/CoreBundle/src/Command/Bundle/InstallCommand.php
+++ b/bundles/CoreBundle/src/Command/Bundle/InstallCommand.php
@@ -22,6 +22,7 @@ use Pimcore\Extension\Bundle\PimcoreBundleManager;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Command\Command;
 
 /**
  * @internal
@@ -51,7 +52,7 @@ class InstallCommand extends AbstractBundleCommand
 
         if ($this->bundleManager->isInstalled($bundle)) {
             $this->io->success(sprintf('Bundle "%s" is already installed', $bundle->getName()));
-            return 0;
+            return Command::SUCCESS;
         }
 
         // sets up installer with console output writer
@@ -70,6 +71,6 @@ class InstallCommand extends AbstractBundleCommand
             $this->getApplication()->getKernel()->getEnvironment()
         );
 
-        return 0;
+        return Command::SUCCESS;
     }
 }

--- a/bundles/CoreBundle/src/Command/Bundle/InstallCommand.php
+++ b/bundles/CoreBundle/src/Command/Bundle/InstallCommand.php
@@ -49,6 +49,11 @@ class InstallCommand extends AbstractBundleCommand
     {
         $bundle = $this->getBundle();
 
+        if (!$this->bundleManager->canBeInstalled($bundle) || $this->bundleManager->isInstalled($bundle)) {
+            $this->io->success(sprintf('Bundle "%s" is already installed', $bundle->getName()));
+            return 0;
+        }
+
         // sets up installer with console output writer
         $this->setupInstaller($bundle);
 

